internal/ingress/types_equals.go

```diff
@@ -105,6 +105,10 @@ func (c1 *Configuration) Equal(c2 *Configuration) bool {
 		return false
 	}
 
+	if len(c1.ControllerPods) != len(c2.ControllerPods) {
+		return false
+	}
+
```

Comment:
This is not enough to detect other updates to controller pods. I realize it is enough for what we want but you're also introducing a new field that we might start using it later on for more than just using length of it.